var/spack/repos/builtin/packages/openssl/package.py

```diff
@@ -76,12 +76,15 @@ class Openssl(Package):   # Uses Fake Autotools, should subclass Package
     version('1.0.1h', sha256='9d1c8a9836aa63e2c6adb684186cbd4371c9e9dcc01d6e3bb447abf2d4d3d093', deprecated=True)
     version('1.0.1e', sha256='f74f15e8c8ff11aa3d5bb5f276d202ec18d7246e95f961db76054199c69c1ae3', deprecated=True)
 
-    variant('systemcerts', default=True, description='Use system certificates')
+    variant('certs', default='system',
+            values=('mozilla', 'system', 'none'), multi=False,
```

Comment:
Should we allow both certs?

Reply:
It would clash with the bundled certificate on some distro's, e.g. fedora:

```
$ docker run fedora:34 /bin/sh -c 'dnf install -y openssl &> /dev/null && stat /etc/pki/tls/cert.pem'
  File: /etc/pki/tls/cert.pem -> /etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem
```

I can test if we can use an arbitrary name in openssl's config dir like `ca-certificates-mozilla.pem`, but I think it only considers `certs.pem` there.

Reply:
Yeah, it seems openssl has two modes, by file or by dir, and if by file, the path hard a hard-coded default: 

```
#  define X509_CERT_FILE          OPENSSLDIR "/cert.pem"
```

it's not globbing *.pem.

Reply:
maybe pem files can be extracted, but that's a bit beyond the scope for me right now... and it's also not to bad to have a single bundle file on slow hpc systems :upside_down_face: